src/_includes/components/header-nav.njk

```diff
@@ -20,7 +20,7 @@
 <div class="nav-smallScreen">
 	<nav class="starter-nav">
 		<button id="menuToggleButton" aria-expanded="false">
-		  <svg><use xlink:href="#menu-icon" /></svg>&nbsp;Menu
+		  <svg><use xlink:href="#menu-icon" /></svg>&nbsp;
```

Comment:
Removing the text "Menu" will introduce an accessibility issue that this menu icon will not be recongnized as a menu button by ATs. It's probably better to wrap the "menu" text in a div or span and show/hide it when needed.